src/main/java/org/spongepowered/api/statistic/Statistics.java

```diff
@@ -24,7 +24,7 @@
  */
 package org.spongepowered.api.statistic;
 
-import org.spongepowered.api.util.generator.dummy.DummyObjectProvider;
+import static org.spongepowered.api.util.generator.dummy.DummyObjectProvider.createFor;
```

Comment:
Don't static import the method `createFor` please. Yes, it does add to line length, but it's clearer when reading where it's coming from.